examples/CockroachEFCoreExample.sln

```diff
@@ -7,6 +7,7 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "CockroachEFCoreExample", "CockroachEFCoreExample.csproj", "{3C29EC49-D49E-454E-A54C-303B6DBCFD4D}"
 EndProject
+doEndProject
```

Comment:
just curious, what's doEndproject?

Reply:
This file is autogenerated via dotnet clean. I don't really know.